src/plugin-main.cpp

```diff
@@ -0,0 +1,88 @@
+/*
+Plugin Name
+Copyright (C) 2024 Mark Jones <mark.jones1112+nospam@gmail.com>
+
+This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation; either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along
+with this program. If not, see <https://www.gnu.org/licenses/>
+*/
+
+#include <obs-module.h>
+#include <plugin-support.h>
+#include <QMainWindow>
+#include <QWidget>
+#include <QDockWidget>
+#include <QString>
+#include <obs-frontend-api.h>
+#include <obs.hpp>
+#include <filesystem>
+#include <util/util.hpp>
+#include <util/platform.h>
+#include "obs-text-mustache-definitions.hpp"
+#include "variables.hpp"
+#include "plugin-lifetime.hpp"
+#include "types.h"
+
+OBS_DECLARE_MODULE()
+OBS_MODULE_USE_DEFAULT_LOCALE(PLUGIN_NAME, "en-US")
+
+bool obs_module_load()
+{
+	InitOBSText();
+	auto *window = (QMainWindow *)obs_frontend_get_main_window();
+
+	obs_frontend_push_ui_translation(obs_module_get_string);
+
+	char *configPath = obs_module_config_path("");
+	try {
+		if(!std::filesystem::create_directory(configPath)) {
+			blog(LOG_DEBUG, "obs-text-mustache-definitions data directory already exists, continuing.");
+		}
+	} catch(std::exception){
+		blog(LOG_INFO, "obs-text-mustache-definitions data directory could not be created.");
+	}
+
+	bfree(configPath);
+
+	auto *obsTextMustache = new OBSTextMustacheDefinitions(window);
+
+	//const QString title = QString::fromUtf8(obs_module_text("Text Template Values"));
+	const QString title = QString::fromUtf8(obs_module_text("OBSTextMustacheDefinitions"));
```

Comment:
would it make sense to move name above title and then do

const QString title = QString::fromUtf8(obs_module_text(name));

and delete line 58 since no longer used?

Reply:
Good catch!  I've updated the code with this.